internal/builders/container/README.md

```diff
@@ -246,7 +246,7 @@ generated as an [in-toto](https://in-toto.io/) statement with a SLSA predicate.
     "builder": {
       "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v1.1.1"
     },
-    "buildType": "https://github.com/slsa-framework/slsa-github-generator/generic@v1",
+    "buildType": "https://github.com/slsa-framework/slsa-github-generator/container@v1",
```

Comment:
the container builder and this one will have the same type, correct?
Do we want to align it with the Dockerfile-based builder TDD? I think the doc used `https://github.com/slsa-framework/slsa-github-generator/docker-based-builder@v1`, but the one you have LGT: as well.

@asraa wdut?

Reply:
I think they can have separate buildTypes for now. We could maybe revisit how these should work and maybe combine them when we support SLSA v1.0.

Reply:
I'm giving the generic container generator the name 'container' here. The convention we have been using so far is to use the same name as the directory name for the builder in `internal/builders`.

Reply:
I'm curious to get @asraa's thoughts too though.

Reply:
> I think they can have separate buildTypes for now. We could maybe revisit how these should work and maybe combine them when we support [SLSA v1.0](https://github.com/MarkLodato/slsa/blob/provenance-refactor/docs/provenance/v1.0.proto).

It seems like the 1.0 spec will make it so that buildType will be agnostic of the environment that it was run in. I'm fine to have it be the same convention as the builder, but part of me thinks that in 1.0 these *should* be the same: the type of "build" performed here was a "no-op" observation, and mostly fills the details on the attestor/github environment, and not much about the process the artifact was built.

Reply:
Agreed, I think that's likely the direction we'll want to go. I'm mostly doing this to cover bases before a GA of the generic container workflow because we'll likely want to do that before SLSA v1.0 gets finalized.

Reply:
That's good with me!

Reply:
Mostly because no one really is relying on this buildType field, and I think it'll be fine to move over -- people know SLSA isn't 1.0 stable yet